### DISCUSSION TITLE
add option to render link tag conditionally

### DISCUSSION THREAD

**Discussion by tomer-yechiel** [remix-run/remix#4640] [Proposals]
Hi,
I didn't see currently an option to load CSS files conditionally, the use case is to load different CSS files if the user local is RTL/LTR.

what do you think about adding to the Links component a new prop filter?
<Links filterBy={(link:LinkDescriptor) => false} />

I can open a PR
thanks!

**Comment by sergiodxa** [comment]
If you can use a media query to detect if the user needs RTL/LTR styles, you can use the `media` option in the links.

If there's no way to do that, you can [Remix Util's DynamicLinks](https://github.com/sergiodxa/remix-utils#dynamiclinks) which receives the data from the loader where you can return if you need RTL or LTR styles.
